lib/assessment.js

```diff
@@ -5,7 +5,18 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  if (!arguments[0]) {
+    return undefined;
+  }
```

Comment:
Good thinking, but actually unnecessary in this case; you're setting `smallest` to `arguments[0]` anyway, so even if no arguments are passed in, you're already returning `undefined`